pkgs/by-name/ne/neocities/package.nix

```diff
@@ -10,10 +11,12 @@ bundlerApp {
   passthru.updateScript = bundlerUpdateScript "neocities";
 
   meta = with lib; {
-    description = "The Neocities CLI, written in Ruby";
+    description = "Neocities Gem - A CLI and library for using the Neocities web site API";
```

Comment:
Small nit:

`meta.description` must: "Not start with the package name" 
> https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

Reply:
I've revised it to read "cli and library for interacting with the Neocities API" -- is this okay?

Reply:
I believe it is